src/BaseTemplates/WebAPI/web.config

```diff
@@ -9,6 +9,6 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
     </handlers>
-    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" startupTimeLimit="3600" stdoutLogEnabled="false"/>
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false"/>
```

Comment:
Also, you need to update the vstemplate files in under src\BaseTemplates to move the web.config out of the wwwroot folder and into the root folder. (https://github.com/aspnet/Templates/blob/dev/src/BaseTemplates/EmptyWeb.vstemplate#L26) I'd leave the empty folder element so the folder gets created when the template is instantiated